content/5_Plat/DP_Ranges.mdx

```diff
@@ -11,27 +11,123 @@ frequency: 2
 
 
 export const problems = {
+	sample: new Problem(
+		'SAPO',
+		'2015 - Space Jazz',
+		'https://saco-evaluator.org.za/cms/sapo2015z/tasks/jazz/description',
+		'Easy',
+		false,
+		['Range DP'],
+		'@below'
+	),
 	general: [
 		new Problem('Gold', '248', '647', 'Easy', false, ['Range DP']),
 		new Problem('CSES', 'Empty String', '1080', 'Normal', false, ['Range DP']),
 		new Problem('CF', 'Zuma', 'problemset/problem/607/B', 'Normal', false, [
 			'Range DP',
 		]),
+		new Problem(
+			'SAPO',
+			'2014 - The Stables of Genghis Khan',
+			'https://saco-evaluator.org.za/cms/sapo2014z/tasks/genghis/description',
+			'Normal',
+			false,
+			['Range DP'],
+			''
+		),
+		new Problem(
+			'Baltic OI',
+			'2009 - Beetle',
+			'https://cses.fi/107/list/',
+			'Hard',
+			false,
+			['Range DP'],
+			''
+		),
 		new Problem('Plat', 'Greedy Pie Eaters', '972', 'Hard', false, [
 			'Range DP',
 		]),
 		new Problem('Plat', 'Subsequence Reversal', '698', 'Hard', false, [
 			'Range DP',
 		]),
 		new Problem('Plat', 'Lightsout', '602', 'Very Hard', false, ['Range DP']),
+		new Problem(
+			'CEOI',
+			'2012 - Sailing Race',
+			'https://oj.uz/problem/view/CEOI12_race',
+			'Very Hard',
+			true,
+			['Range DP', 'convex'],
```

Comment:
> 'convex'

wait what

Reply:
Well not exactly convex - more like one of the DP arrays is monotone (which you use to optimize from O(N^4) to O(N^3)), but no other problem has the "Monotone" tag

Reply:
I don't think it's necessary to add a tag for this.